homeassistant/components/vulcan/calendar.py

```diff
@@ -29,8 +33,8 @@ async def async_setup_entry(
     config_entry: ConfigEntry,
     async_add_entities: AddEntitiesCallback,
 ) -> None:
-    """Set up the calendar platform for event devices."""
-    VulcanCalendarData.MIN_TIME_BETWEEN_UPDATES = timedelta(
+    """Set up the calendar platform for entity."""
+    VulcanCalendarEntity.MIN_TIME_BETWEEN_UPDATES = timedelta(
         minutes=config_entry.options.get(CONF_SCAN_INTERVAL, DEFAULT_SCAN_INTERVAL)
```

Comment:
Side note: We shouldn't allow the user to change scan interval like this. We support a system option to disable polling completely per integration and then the user can automate updates with the entity service `homeassistant.update_entity`.

https://www.home-assistant.io/integrations/homeassistant/#service-homeassistantupdate_entity